gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/DnsExample.java

```diff
@@ -336,7 +337,7 @@ public void run(Dns dns, String... args) {
           ChangeRequest change = iterator.next();
           System.out.printf("%nID: %s%n", change.generatedId());
           System.out.printf("Status: %s%n", change.status());
-          System.out.printf("Started: %s%n", FORMATTER.format(change.startTimeMillis()));
+          System.out.printf("Started: %s%n", FORMATTER.print(change.startTimeMillis()));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [System.out.printf is used](https://www.codacy.com/app/mziccard/gcloud-java/file/2353572075/issues/source?bid=3123103&fileBranchId=3217862#l340)